docs/source/cpp/tables.rst

```diff
@@ -82,7 +82,7 @@ and computation functions, possibly incremental.
          structure as described in text above.
 
 Record batches can be sent between implementations, such as via 
-:ref:`IPC <format-ipc>` or
+:ref:`IPC format <format-ipc>` or
```

Comment:
Is this really needed?
https://arrow.apache.org/docs/cpp/tables.html#record-batches renders correctly without this:

> Record batches can be sent between implementations, such as via [IPC](https://arrow.apache.org/docs/format/Columnar.html#format-ipc) or via ...